projects/storefrontapp-e2e-cypress/cypress/helpers/customer-ticketing/cutsomer-ticketing.ts

```diff
@@ -0,0 +1,38 @@
+import { standardUser } from '../../sample-data/shared-users';
+import { generateMail, randomString } from '../user';
+import { clickHamburger } from '../checkout-flow';
+
+export function loginRegisteredUser() {
+  standardUser.registrationData.email = generateMail(randomString(), true);
+  cy.requireLoggedIn(standardUser);
+  cy.reload();
+}
+
+export function openMyAccountDropDown(){
+  cy.visit('/');
+  cy.wait(5000);
+  cy.onMobile(() => {
+    clickHamburger();
+  });
+  cy.get('.accNavComponent button').click();
+}
+
+export function clickCustomerSupport(){
+  cy.get('.accNavComponent li:nth-child(14)').should('contain.text', 'Customer Service');
+  cy.get('.accNavComponent li:nth-child(14) a').click();
+}
+
+export function verifyCustomerSupportPageVisit(){
```

Comment:
```suggestion
export function verifyTicketListingPageVisit(){
```
to avoid confusion with other customer support pages

Reply:
Done.